docs/cli/completion.md

```diff
@@ -0,0 +1,86 @@
+# Completions
+
+Completions make it a little easier to run specdown, allowing you to press tab and have a half typed command be completed. Completions can be generated in specdown by running the completion command.
+
+## Example
+
+### Bash
+We support assorted shells 
+
+```shell,script(expected_exit_code=0)
+specdown completion Bash
```

Comment:
I'm not sure about the capitalisation of `Bash` here? The implementation suggests that possible values are lowercase (are they case insensitive?). On a personal level, I'd prefer all lowercase unless there's a good reason?

Reply:
There wasn't other than I tried an enum first and it only let me do them as capitalised them, I'll take them out